server/api/eventinfo/eventinfo.controller.js

```diff
@@ -71,9 +71,7 @@ function removeEntity(res) {
 function saveEventInfoImage(files, fields, cb){
   var imagePaths = [],
       asyncTasks = [];
-  if (!files){
-    return imagePaths;
-  }
+  if (!files || !files.files){return imagePaths;}
```

Comment:
if it's a single line statement, stylistically I think it should not have brackets :penguin: